src/Raven.Client/Documents/Session/DocumentQuery.cs

```diff
@@ -1213,6 +1213,7 @@ public IRavenQueryable<T> ToQueryable()
 
             var queryStatistics = new QueryStatistics();
             var highlightings = new LinqQueryHighlightings();
+            IncludeTimings(out _);
```

Comment:
You are doing this unconditionally, which change the meaning, not all queries with `ToQueryable` will have `Timings`, even if they aren't actually in use.

Reply:
Fixed.